nomad/dev/Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL = bash
+
+ENV ?= dev
+
+env.auto.tfvars:
+	@echo 'hcloud_token = "$(HCLOUD_TOKEN)"' >> "$@"
+
+.terraform:
+	tofu init
+
+validate: .terraform
+	tofu validate
+
+up: .terraform env.auto.tfvars
+	tofu apply -auto-approve
+	$(MAKE) port-forward
+
+down: .terraform env.auto.tfvars
+	if test -f files/registry-port-forward.sh; then files/registry-port-forward.sh down; fi
+	tofu destroy -auto-approve
+
+port-forward:
+	files/registry-port-forward.sh up
+
+clean:
+	rm -Rf files/ .terraform/ terraform.tfstate* env.auto.tfvars
+
+deps: clean
+	sed -i -e 's|source = ".*"|source = "./.."|' main.tf
+	tofu init -upgrade
```

Comment:
This is only required in the `nomad-dev-env/example` dir. The target `deps` can be removed here.